### PR TITLE
docs: add English version of OpenMLDB QuickStart documentation

### LINKED ISSUE #1321: docs: translate OpenMLDB QuickStart from Chinese to English
1. Translate this latest OpenMLDB QuickStart from Chinese to English: https://docs.openmldb.ai/quickstart/openmldb_quickstart
The source of this doc is here: https://github.com/4paradigm/openmldb-docs-zh/blob/dev/quickstart/openmldb_quickstart.md
2. Save the translated English quickstart in this repo as docs/en/openmldb_quickstart.md

### PR DESCRIPTION
**Pull request by cswjd**
feat: add English version of OpenMLDB QuickStart documentation
close #1321

### CODE REVIEW

**Comment by cswjd** [comment]
fixes #1321

File: docs/en/openmldb_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,379 @@
+# OpenMLDB Quick Start
+
+This tutorial provides a quick start guide for OpenMLDB. By establishing a database, importing data, offline feature calculation, SQL solution online, and online real-time feature calculation, the basic usage process of the standalone version of OpenMLDB and the cluster version of OpenMLDB is demonstrated.
```
Comment (vagetablechicken) [inline docs/en/openmldb_quickstart.md]: basic steps are: create database, ..., deploy SQL, online ...

File: docs/en/openmldb_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+
+## 1. Environment and data preparation
+
+> :warning: docker engine version required >= 18.03
+
+This tutorial is developed and deployed based on the OpenMLDB CLI, so first you need to download the sample data and start the OpenMLDB CLI. We recommend using the prepared docker image for a quick experience
```
Comment (vagetablechicken) [inline docs/en/openmldb_quickstart.md]: missing the full stop

File: docs/en/openmldb_quickstart.md [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+
+```sql
+> CREATE DATABASE demo_db;
+> USE demo_db;
+> CREATE TABLE demo_table1(c1 string, c2 int, c3 bigint, c4 float, c5 double, c6 timestamp, c7 date);
+```
+
+#### 2.2.2 Data preparation
+
+Import the previously downloaded sample data (download the saved data in [1.2 Sample Data](#1.2-Sample Data)) as training data for offline and online feature calculation.
```
Comment (vagetablechicken) [inline docs/en/openmldb_quickstart.md]: what means 'download the saved data'?

File: docs/en/openmldb_quickstart.md (hunk 1 above)
Comment (vagetablechicken) [reply docs/en/openmldb_quickstart.md]: should we replace all `online/offline calculation` with `online/offline extraction`? @lumianph

Comment (cswjd) [reply docs/en/openmldb_quickstart.md]: ok

File: docs/en/openmldb_quickstart.md (hunk 2 above)
Comment (cswjd) [reply docs/en/openmldb_quickstart.md]: ok

File: docs/en/openmldb_quickstart.md (hunk 3 above)
Comment (cswjd) [reply docs/en/openmldb_quickstart.md]: changed to: the saved data in...

File: docs/en/openmldb_quickstart.md (hunk 1 above)
Comment (cswjd) [reply docs/en/openmldb_quickstart.md]: @lumianph  confirms that extraction is better.

**Review by lumianph — APPROVED** [review]

File: docs/en/openmldb_quickstart.md [hunk 4]
```diff
@@ -0,0 +1,379 @@
...
+The following is the expected return result for this query (the computed features are stored in the `data` field):
+
+```json
+{"code":0,"msg":"ok","data":{"data":[["aaa",11,22]]}}
+
+```
+
+#### 3.3.8 Result description of real-time feature extraction
+
+Real-time request (deployment execution) is SQL execution in request mode (request mode). Unlike batch mode (batch mode), request mode will only perform SQL extractions on request rows. In the previous example, the POST input is used as the request line, assuming this line of data exists in the table demo_table1, and execute SQL on it:
```
Comment (vagetablechicken) [inline docs/en/openmldb_quickstart.md]: `request mode (request mode)`?  Plz remove the brackets.

File: docs/en/openmldb_quickstart.md [hunk 5]
```diff
@@ -0,0 +1,379 @@
...
+```
+SELECT c1, c2, sum(c3) OVER w1 AS w1_c3_sum FROM demo_table1 WINDOW w1 AS (PARTITION BY demo_table1.c1 ORDER BY demo_table1.c6 ROWS BETWEEN 2 PRECEDING AND CURRENT ROW);
+```
+The specific extraction logic is as follows (the actual extraction will be optimized to reduce the amount of extraction):
```
Comment (vagetablechicken) [inline docs/en/openmldb_quickstart.md]: reduce the amount of extraction -> reduce the computation

File: docs/en/openmldb_quickstart.md [hunk 6]
```diff
@@ -0,0 +1,379 @@
...
+1. According to the request line and the `PARTITION BY` partition of the window, filter out the lines whose c1 is "aaa", and sort them according to c6 from small to large. So theoretically, the intermediate data table after partition sorting is shown in the following table. Among them, the first row after the request behavior is sorted.
```
Comment (vagetablechicken) [inline docs/en/openmldb_quickstart.md]: the `PARTITION BY` in window clause

File: docs/en/openmldb_quickstart.md [hunk 7]
```diff
@@ -0,0 +1,379 @@
...
+```
+ ----- ---- ---- ---------- ----------- --------------- - -------------
+  c1 c2 c3 c4 c5 c6 c7
+ ----- ---- ---- ---------- ----------- --------------- - -------------
+  aaa 11 22 1.2 1.3 1635247427000 2021-05-20
+  aaa 12 22 2.200000 12.300000 1636097890000 1970-01-01
+  aaa 11 22 1.200000 11.300000 1636097290000 1970-01-01
+ ----- ---- ---- ---------- ----------- --------------- - -------------
+```
+2. The window range is `2 PRECEDING AND CURRENT ROW`, so we cut out the real window in the above table, the request row is the smallest row, the previous 2 rows do not exist, but the window contains the current row, so the window has only Request line this line.
```
Comment (vagetablechicken) [inline docs/en/openmldb_quickstart.md]: Request line this line -> one row(the request row)

File: docs/en/openmldb_quickstart.md (hunk 4 above)
Comment (cswjd) [reply docs/en/openmldb_quickstart.md]: thank you.

File: docs/en/openmldb_quickstart.md (hunk 6 above)
Comment (cswjd) [reply docs/en/openmldb_quickstart.md]: ok, thank you.

File: docs/en/openmldb_quickstart.md (hunk 7 above)
Comment (cswjd) [reply docs/en/openmldb_quickstart.md]: thank you.

File: docs/en/openmldb_quickstart.md (hunk 5 above)
Comment (cswjd) [reply docs/en/openmldb_quickstart.md]: ok, thank you.

**Review by lumianph — APPROVED** [review]

**Comment by lumianph** [comment]
@vagetablechicken pls help review and confirm the revision

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**